include/zephyr/drivers/video-controls.h

```diff
@@ -107,18 +227,307 @@ enum video_power_line_frequency {
+ */
+#define VIDEO_CID_AUTO_N_PRESET_WHITE_BALANCE (VIDEO_CID_CAMERA_CLASS_BASE + 20)
+enum video_auto_n_preset_white_balance {
+	VIDEO_WHITE_BALANCE_MANUAL = 0,
+	VIDEO_WHITE_BALANCE_AUTO = 1,
+	VIDEO_WHITE_BALANCE_INCANDESCENT = 2,
+	VIDEO_WHITE_BALANCE_FLUORESCENT = 3,
+	VIDEO_WHITE_BALANCE_FLUORESCENT_H = 4,
+	VIDEO_WHITE_BALANCE_HORIZON = 5,
+	VIDEO_WHITE_BALANCE_DAYLIGHT = 6,
+	VIDEO_WHITE_BALANCE_FLASH = 7,
+	VIDEO_WHITE_BALANCE_CLOUDY = 8,
+	VIDEO_WHITE_BALANCE_SHADE = 9,
+	VIDEO_WHITE_BALANCE_GREYWORLD = 10,
+};
+
+/** Enables or disables the camera's wide dynamic range feature.
+ * This feature allows to obtain clear images in situations where intensity of the illumination
+ * varies significantly throughout the scene, i.e. there are simultaneously very dark and very
+ * bright areas. It is most commonly realized in cameras by combining two subsequent frames with
+ * different exposure times.
+ */
+#define VIDEO_CID_WIDE_DYNAMIC_RANGE (VIDEO_CID_CAMERA_CLASS_BASE + 21)
+
+/** Enables or disables image stabilization. */
+#define VIDEO_CID_IMAGE_STABILIZATION (VIDEO_CID_CAMERA_CLASS_BASE + 22)
+
+/** Determines the ISO equivalent of an image sensor indicating the sensor's sensitivity to light.
+ * The numbers are expressed in linear scale. Applications should interpret the values as standard
+ * ISO values multiplied by 1000, e.g. control value 800 stands for ISO 0.8. Drivers will usually
+ * support only a subset of standard ISO values.
+ * Setting this control when @ref VIDEO_CID_ISO_SENSITIVITY_AUTO is not
+ * @ref VIDEO_ISO_SENSITIVITY_MANUAL is undefined.
+ * Drivers should ignore such requests.
+ */
+#define VIDEO_CID_ISO_SENSITIVITY (VIDEO_CID_CAMERA_CLASS_BASE + 23)
+
+/** Enables or disables automatic ISO sensitivity adjustments. */
+#define VIDEO_CID_ISO_SENSITIVITY_AUTO (VIDEO_CID_CAMERA_CLASS_BASE + 24)
+enum video_iso_sensitivity_auto {
+	VIDEO_ISO_SENSITIVITY_MANUAL = 0,
+	VIDEO_ISO_SENSITIVITY_AUTO = 1,
+};
+
+/** Determines how the camera measures the amount of light available for the frame exposure. */
+#define VIDEO_CID_EXPOSURE_METERING (VIDEO_CID_CAMERA_CLASS_BASE + 25)
+enum video_exposure_metering {
+	VIDEO_EXPOSURE_METERING_AVERAGE = 0,
+	VIDEO_EXPOSURE_METERING_CENTER_WEIGHTED = 1,
+	VIDEO_EXPOSURE_METERING_SPOT = 2,
+	VIDEO_EXPOSURE_METERING_MATRIX = 3,
+};
+
+/** This control selects scene programs optimized for common shooting scenes.
+ * Within these modes the camera determines best exposure, aperture, focusing, light metering,
+ * white balance and equivalent sensitivity. The controls of those parameters are influenced by
+ * the scene mode control. An exact behavior in each mode is subject to the camera specification.
+ * When the scene mode feature is not used, this control should be set to
+ * @ref VIDEO_SCENE_MODE_NONE to make sure the other possibly related controls are accessible.
+ */
+#define VIDEO_CID_SCENE_MODE (VIDEO_CID_CAMERA_CLASS_BASE + 26)
```

Comment:
REMOVE? Used in only two Linux driver:
`drivers/media/i2c/s5c73m3/s5c73m3-ctrls.c`